tests/test_parser.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+import imod_coupler.parser
+from imod_coupler import __version__
+
+
+def test_get_version() -> None:
+    with pytest.raises(SystemExit) as cm:
+        output_version = imod_coupler.parser.parse_args(["--version"])
+        assert cm.value.code == 0
+        assert output_version is not None
+        assert output_version == __version__
```

Comment:
You expect an exception AND add assertions?
Are these assertions even checked then, or is the exception thrown when calling `parse_args`?

Reply:
The thing is that the ArgumentParser, when performing the `version` action, actually also exits the whole program and that can be caught with a SystemExit. See: https://github.com/python/cpython/blob/c8cf5d7d148944f2850f25b02334400dd0238cb0/Lib/argparse.py#L2637C1-L2643C26

The test was not working, so I have updated the assertions to be outside of the `with` clause.